src/5e-SRD-Spells.json

```diff
@@ -541,13 +543,14 @@
       "Dispel Magic.",
       "Spells and magical effects such as dispel magic have no effect on the sphere. Likewise, the spheres created by different antimagic field spells don't nullify each other."
     ],
-    "range": "Self",
+    "range": {"value": 0, "unit": "ft"},
+    "range_type": "self",
     "components": ["V", "S", "M"],
     "material": "A pinch of powdered iron or iron filings.",
     "ritual": false,
-    "duration": "Up to 1 hour",
+    "duration": {"value": 1, "unit": "hrs", "up-to": true},
```

Comment:
Typical attribute names in the database use underscores to delimit words. `up-to` should probably be `up_to`, or maybe just `upto`.

Reply:
upto can be confusing, opting for up_to instead